src/main/java/org/gridsuite/study/server/StudyService.java

```diff
@@ -343,6 +343,33 @@ public StudyService(
         });
     }
 
+    public Mono<Collection<String>> getExportFormats() {
+        String path = UriComponentsBuilder.fromPath(DELIMITER + NETWORK_CONVERSION_API_VERSION + "/export/formats")
+                .toUriString();
+
+        ParameterizedTypeReference<Collection<String>> typeRef = new ParameterizedTypeReference<Collection<String>>() { };
+
+        return webClient.get()
+                .uri(networkConversionServerBaseUri + path)
+                .retrieve()
+                .bodyToMono(typeRef);
+    }
+
+    public Mono<ResponseEntity<byte[]>> exportNetwork(String studyName, String format) {
```

Comment:
Don't use ResponseEntity in the StudyService. ResponseEntity is about http headers, it should only appear in the StudyController.